barretenberg/cpp/src/barretenberg/ultra_honk/merge_prover.hpp

```diff
@@ -12,31 +12,28 @@ namespace bb::honk {
 /**
  * @brief Prover class for the Goblin ECC op queue transcript merge protocol
  *
- * @tparam Flavor
  */
-template <typename Flavor> class MergeProver_ {
-    using FF = typename Flavor::FF;
-    using Polynomial = typename Flavor::Polynomial;
-    using CommitmentKey = typename Flavor::CommitmentKey;
-    using Commitment = typename Flavor::Commitment;
-    using PCS = typename Flavor::PCS;
-    using Curve = typename Flavor::Curve;
+class MergeProver {
+    using Curve = curve::BN254;
```

Comment:
shouldn't still be taken from the flavor, even though `GoblinUltra` flavor is going to be the fixed one